### PR TITLE
[SD-3712] - Mark the stories to send an sms once published

### PR DESCRIPTION
**Pull request by akintolga**

### CODE REVIEW

**Comment by akintolga** [comment]
Depends on https://github.com/superdesk/superdesk-core/pull/251

File: scripts/superdesk-authoring/authoring.js [hunk 1]
```diff
@@ -72,6 +73,7 @@
         company_codes: {},
         ednote: {},
         headline: {maxlength: 64},
+        sms: {maxlength: 64},
```
Comment (mdhaman) [inline scripts/superdesk-authoring/authoring.js]: i think this should be 160 or whatever is max sms message length.

**Comment by marwoodandrew** [comment]
:+1:

**Comment by ioanpocol** [comment]
:+1:

**Pull request merged**